src/components/icons/Copy.tsx

```diff
@@ -7,7 +7,7 @@ export const Copy = ({ title = 'Copy', size = defaultIconProps.size, color }: SV
 
   return (
     <Icon title={title} lineHeight="0">
-      <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <svg width="14" height="14" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
you can replace these with `width={size} height={size}` so that size can be adjusted via a prop